powersimdata/data_access/data_access.py

```diff
@@ -27,6 +30,29 @@ def __init__(self, root):
         self.root = root
         self.join = fs2.path.join
 
+    def read(self, filepath):
+        """Reads data from local machine.
+
+        :param str filepath: path to file, with extension either 'pkl', 'csv', or 'mat'.
+        :return: (*pandas.DataFrame* or *dict*) -- pkl and csv files will be returned as
+            a data frame, while a mat file will be returned as a dictionary
```

Comment:
I'm not 100% sure about the return type - e.g. if we load a change table from .pkl, would we return a `ChangeTable` instance? I'd say if there is no static guarantee on the type, best we could do is `object` (with some description about the possible values).

Reply:
Unpickling a change table returns a `dict`.